exchanges/deribit/deribit_test.go

```diff
@@ -712,6 +712,53 @@ func TestWSRetrieveLastTradesByInstrumentAndTime(t *testing.T) {
 	}
 }
 
+func TestWSProcessTrades(t *testing.T) {
+	t.Parallel()
+
+	d := new(Deribit) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
+	require.NoError(t, testexch.Setup(d), "Setup instance must not error")
+	testexch.FixtureToDataHandler(t, "testdata/wsAllTrades.json", d.wsHandleData)
+	close(d.Websocket.DataHandler)
+
+	p, a, err := d.getAssetPairByInstrument("BTC-PERPETUAL")
+	require.NoError(t, err, "getAssetPairByInstrument must not error")
+
+	exp := []trade.Data{
+		{
+			Exchange:     d.Name,
+			CurrencyPair: p,
+			Timestamp:    time.UnixMilli(1742627465811).UTC(),
+			Price:        84295.5,
+			Amount:       8430.0,
+			Side:         order.Buy,
+			TID:          "356130997",
+			AssetType:    a,
+		},
+		{
+			Exchange:     d.Name,
+			CurrencyPair: p,
+			Timestamp:    time.UnixMilli(1742627361899).UTC(),
+			Price:        84319.0,
+			Amount:       580.0,
+			Side:         order.Sell,
+			TID:          "356130979",
+			AssetType:    a,
+		},
+	}
+	require.Len(t, d.Websocket.DataHandler, 2, "Must see the correct number of trades")
```

Comment:
```suggestion
	require.Len(t, d.Websocket.DataHandler, len(exp), "Must see the correct number of trades")
```

Just so it scales when more added?

Reply:
done 👍🏼